docs/software/globus-mass-update-procedure.md

```diff
@@ -0,0 +1,185 @@
+Globus mass update procedure
+============================
+
+Globus consists of many packages, which we tend to update at the same time. This requires extra work, primarily to prevent dependency issues.
+
+Prep work
+---------
+
+### Docs
+
+Create a spreadsheet or table of the builds. Table should have NVR, perhaps URL, status (not started, imported, built, tested), and comments (mostly to record if it was a simple pass-through or not).
+
+Get packages to update, using `osg-outdated-epel-pkgs` from `opensciencegrid/tools`.
+
+To get in N-V-R format:
+
+``` console
+[you@host]$ ./osg-outdated-epel-pkgs | \
+    egrep '^(globus|myproxy|gsi)' | \
+    awk 'BEGIN {OFS=""} {print $1, "-", $3}'
+```
+
+or to split up N and V-R in a comma-separated way (which you can feed into a Google Sheet to turn it into two columns):
+
+``` console
+[you@host]$ ./osg-outdated-epel-pkgs | \
+    egrep '^(globus|myproxy|gsi)' | \
+    awk 'BEGIN {OFS=""} {print $1, ",", $3}'
+```
+
+### SVN
+
+Create a separate SVN branch and populate it with all the packages you will update. (Get the list from the doc created above).
+
+``` console
+[you@uw]$ svn mkdir file:///p/vdt/workspace/svn/native/redhat/branches/globus
+### From a checkout, in native/redhat
+[you@uw]$ for x in %RED%<PACKAGES>%ENDCOLOR%; do \
+     svn copy $x branches/globus/${x#trunk/}; \
+   done
+```
+
+### Koji (Mat/Carl)
+
+This requires a Koji administrator. Koji admins as of August 2017 are Mat Selmeci and Carl Edquist.
+
+Ensure Koji tags exist: a destination tag, and a build tag, one for each dver, e.g.:
+
+-   el6-globus
+-   el6-globus-build
+-   el7-globus
+-   el7-globus-build
+
+Set up tag inheritence: base the build tags off of the corresponding `dist-el?-build` tag. This is because we don't want old osg packages interfering with the new versions we're building. These may already exist -- check the `el?-globus-build` tags in the web interface.
+
```

Comment:
If they already exist, then what? Can the dev skip the command immediately below this?